Formula/azure-cli.rb

```diff
@@ -3,9 +3,9 @@ class AzureCli < Formula
 
   desc "Microsoft Azure CLI 2.0"
   homepage "https://docs.microsoft.com/cli/azure/overview"
-  url "https://codeload.github.com/Azure/azure-cli/legacy.tar.gz/adb59a62ecbbea5e7c7bddd5e062ef7806825d02"
-  version "2.16.0"
-  sha256 "dc8318bef61fb1054d9a64f6243b203438d8c66114c82879339aae568460ff58"
+  url "https://codeload.github.com/Azure/azure-cli/legacy.tar.gz/57ce1e6689dcbdb21f62bae6f520e74e95c0c4e0"
```

Comment:
@azclibot can we update this url to `https://github.com/Azure/azure-cli/archive/azure-cli-2.17.0.tar.gz`, then we can get rid of version line.

Reply:
Furthermore, I wonder if it would be possible to publish that big source archive to pypi and use that instead. Thanks!

Reply:
Hi @chenrui33, thanks a lot for your feedback. 
Could we keep it now to catch product release schedule and we will make the change in next release?

Reply:
Hi @Juliehzl, I think it's better to change it now, as it might be forgotten if we wait till the next release.

Reply:
@julienXX I will log a ticket on your side (fine with automate the style for the next run) and make the change now.